wsdl/ver10/advancedsecurity/wsdl/advancedsecurity.wsdl

```diff
@@ -839,6 +875,127 @@
 				<xs:anyAttribute processContents="lax"/>
 			</xs:complexType>
 			<!--===============================-->
+			<xs:simpleType name="AuthorizationServerConfigurationType">
+				<xs:restriction base="xs:string">
+					<xs:enumeration value="OAuthAuthorizationCode">
+						<xs:annotation>
+							<xs:documentation>OAuth2 authorization code flow per RFC 6749</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="OAuthClientCredentials">
+						<xs:annotation>
+							<xs:documentation>OAuth2 client credentials grant flow per RFC 6749</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+				</xs:restriction>
+			</xs:simpleType>
+			<!--===============================-->
+			<xs:simpleType name="ClientAuthenticationMethod">
+				<xs:restriction base="xs:string">
+					<xs:enumeration value="client_secret_basic">
+						<xs:annotation>
+							<xs:documentation>Use HTTP Authorization header to specify client_secret per RFC 6749</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="client_secret_post">
+						<xs:annotation>
+							<xs:documentation>Use HTTP POST body to specify client_secret per RFC 6749</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="client_secret_jwt">
+						<xs:annotation>
+							<xs:documentation>Use a HMAC signed JWT using client_secret as shared secret per OpenID Connect Core</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="private_key_jwt">
+						<xs:annotation>
+							<xs:documentation>Use PKI signed JWT using private key per OpenID Connect Core</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="tls_client_auth">
+						<xs:annotation>
+							<xs:documentation>Use PKI certificate to authenticate per RFC 8705</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="self_signed_tls_client_auth">
+						<xs:annotation>
+							<xs:documentation>Use self-signed certificate to authenticate per RFC 8705</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+				</xs:restriction>
+			</xs:simpleType>
+			<!--===============================-->
+			<xs:complexType name="AuthorizationServerConfigurationData">
```

Comment:
This config seems to be missing an entry for KeyID (used by private_key_jwt) and a CertificateId field (used by tls_client_auth and self_signed_tls_client_auth)